api/imp_links.json

```
@@ -28,5 +28,17 @@
       "title": "Newcomers Guide",
       "url": "https://wiki.gnome.org/Newcomers/"
     }
+  ],
+  "52North Spatial Information Research GmbH": [
+    {
+      "title":"Public Repository for the Connected Systems SWG",
+      "url": "https://github.com/52North/ogcapi-connected-systems"
+    }
+  ],
+  "aossie.org":[
```

```suggestion
  "aossie":[
```
I think this should be correct here as we want this to match with the org slug when using `slugify()`.